nanoFirmwareFlasher.Library/appsettings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "iConnectionString": "Put iAppKeyConnectionString here!"
```

Comment:
Can you add a token and adjust the build pipeline to use a token? We can add the secret into the repo.

Reply:
@josesimoes could you add the secret token/connectionstring? I don't have access to this connectionstring.